quiz_app/serializers.py

```diff
@@ -0,0 +1,219 @@
+from rest_framework import serializers
+from .models import Category, Tag, Quiz, Question, Answer, Participant, Feedback
+from django.utils import timezone
+
+
+class CategorySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Category
+        fields = '__all__'
```

Comment:
don't use __all__, define each of the fields that are needed exclude any extra fields.